tests/test_node_mono_repo.py

```diff
@@ -157,6 +157,18 @@ def test_write_release_please_config():
         )
 
 
+def test_copy_quickstart():
+    with util.copied_fixtures_dir(FIXTURES):
```

Comment:
The root `fixtures` directory contains folders that represent different repository structures. For this case, we should put the fixtures in a subfolder that represents the scenario you're testing (rather than in the fixtures root).

Reply:
For this one, I'm following this pattern: https://github.com/googleapis/synthtool/blob/6d304d5520aee517aa8b54c2eb8bd1d9fcca8d0f/tests/test_node_mono_repo.py#L79

Otherwise, I'd need to create a whole new directory with the same fixtures, making the library bigger. What do you think?

Reply:
You are already essentially creating a new test scenario, it's just in the root `fixtures` directory.